test/lib/vimeo_test.js

```diff
@@ -0,0 +1,642 @@
+/* eslint-env mocha */
```

Comment:
This is how `standard` [knows about mocha](https://github.com/standard/standard#what-about-mocha-jasmine-qunit-etc).